### PR TITLE
util/imagetools: remove unused Resolver.ImageConfig

### PR DESCRIPTION
**Pull request by thaJeztah**

### CODE REVIEW

**Comment by thaJeztah** [comment]
oh, nice; that also removes ttrpc as dependency 😄

**Review by crazy-max — APPROVED** [review]

**Comment by thaJeztah** [comment]
@jedevc good to go?

**Review by jedevc — APPROVED** [review]

File: go.mod [hunk 1]
```diff
@@ -76,7 +76,6 @@ require (
 	github.com/cenkalti/backoff/v4 v4.2.0 // indirect
 	github.com/cespare/xxhash/v2 v2.2.0 // indirect
 	github.com/cloudflare/cfssl v0.0.0-20181213083726-b94e044bb51e // indirect
-	github.com/containerd/ttrpc v1.2.2 // indirect
```
Comment (crazy-max) [inline go.mod:79]: Now sure why it does remove ttrpc :thinking:

Comment (crazy-max) [reply go.mod:79]: Oh `github.com/moby/buildkit/util/imageutil` has a depedency on it nevermind.

**Pull request merged**